src/java/main/org/apache/zookeeper/server/ServerStats.java

```diff
@@ -53,10 +53,11 @@ public ServerStats(Provider provider) {
     synchronized public long getMinLatency() {
         return minLatency == Long.MAX_VALUE ? 0 : minLatency;
     }
-
-    synchronized public long getAvgLatency() {
+    
+    synchronized public double getAvgLatency() {
         if (count != 0) {
-            return totalLatency / count;
+            //be account to four decimal places
+            return (totalLatency * 10000 / count) / 10000.0;
```

Comment:
The findbugs is complaining about integral division result, should change it to:

`return (totalLatency * 10000.0 / count) / 10000.0;`

Reply:
@lvfangmin I test it in my local and find no complain（findbugs version 3.0.1）
where can I find the warnings?

Reply:
@maoling here is the link: 

https://builds.apache.org/job/PreCommit-ZOOKEEPER-github-pr-build/2198/artifact/build/test/findbugs/newPatchFindbugsWarnings.html